docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -98,6 +98,36 @@ comprising the number of shards in the index, the total size of the index in
 bytes, and the maximum number of segments per shard in the index. Defaults to
 `false`, meaning that this information is omitted.
 
+`sort`::
+(Optional, string)
+Allows setting a sort order for the result. Defaults to `start_time`, i.e. sorting by snapshot start time stamp.
++
+.Valid values for `sort`
+[%collapsible%open]
+====
+`start_time`::
+  Sort snapshots by their start time stamp and break ties by snapshot name.
+
+`duration`::
+  Sort snapshots by their duration and break ties by snapshot name.
+
+`name`::
+  Sort snapshots by their name.
+
+`indices`::
+  Sort snapshots by the number of indices they contain and break ties by snapshot name.
+====
+
+`size`::
+(Optional, integer)
+ Maximum number of snapshots to return. Defaults to `0` which means return all that match the request without limit.
+
+`after`::
+(Optional, string)
+  Offset from which to start the snapshot listing defined by sort value and snapshot name as a string of the form
+  `<sort_value>,<snapshot_name>`. Defaults to not set which means that listing will start from the first snapshot that matches the
```

Comment:
I wonder if we should introduce the `next` field into the response too, such that users can just copy that to get the next page? Can certainly also go in a follow-up, I find it less important to have initially at least.

Reply:
++ that sounds like a neat idea. Follow-up for that is probably fine, this one is already pretty large :)

Reply:
Hmm `next` sounds more useful than this `after` parameter. Why are we preferring `after`?

Reply:
My understanding was that we would return a `next` field and use it as the `after` parameter? So it would just be a usability enhancement. to this parameter?

Reply:
Ah ok. I would rather that the parameter were more opaque then - if we document how to construct it then folk will do just that and then we'll be stuck supporting it even if, say, we want to support multiple sort keys or some other enhancement later on.

Reply:
Fine with me, shall we just drop the documentation for the parameter in this PR then and document `next` and `after` in a follow-up?

Reply:
Sure, we could even drop this whole parameter until the followup: IMO `LIMIT` has value even without `OFFSET`. Your call, I'm happy either way.